src/utils/enums.js

```diff
@@ -1,11 +1,16 @@
+import { unset } from "lodash";
```

Comment:
Remove this one, It's not used in this file. I guess you mis-import it by line 34...